src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="position">
+    <form
+      action="#"
+      data-qa="big"
+      class="form form__big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        class="big__input"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+
+    <form
+      action="#"
+      data-qa="small"
+      class="form form__small"
```

Comment:
The class name 'form__small' has the same issue as 'form__big' - it represents style rather than content. Consider renaming it to reflect the purpose or content of the form.